### PR TITLE
Add a scan status endpoint

### PR DESCRIPTION
**Pull request by ro-tex**
# PULL REQUEST

## Checklist

- [ ] All git commits are signed. (REQUIRED)
- [ ] All new methods or updated methods have clear docstrings.
- [ ] Testing added or updated for new methods.
- [ ] Verified if any changes impact the WebPortal Health Checks.
- [ ] Appropriate documentation updated.
- [ ] Changelog file created.

### CODE REVIEW

File: scanner/scanner_test.go [hunk 1]
```diff
@@ -1,26 +1,27 @@
-package workers
+package scanner_test
```
Comment (ro-tex) [inline scanner/scanner_test.go:1]: No functional changes.

The only thing changed in this file is the new test package name which is used in order to work around a cyclical dependency with the test package.

File: scanner/scanner.go [hunk 2]
```diff
@@ -1,20 +1,19 @@
-package workers
+package scanner
```
Comment (ro-tex) [inline scanner/scanner.go:1]: The only changed lines in this file are the ones that contain "status". Everything else is the same, just moved around. I'm sorry for the large and complex diff but it really needed some house keeping before we wrap the service up.

File: api/handlers.go [hunk 3]
```diff
@@ -263,6 +264,17 @@ func (api *API) sweepStatusGET(w http.ResponseWriter, _ *http.Request, _ httprou
 	api.WriteJSON(w, api.staticSweeper.Status())
 }
 
+// scannerStatusGET responds with the status of the latest scan.
+func (api *API) scannerStatusGET(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	verbose, _ := strconv.ParseBool(r.FormValue("verbose"))
```
Comment (ChrisSchinnerl) [inline api/handlers.go:269]: I'm usually in favour of forcing the parsing to succeed if an optional flag is specified to help users immediately see potential mistakes they made on their end. e.g. a typo

**Review by MSevey — APPROVED** [review]

**Review by ChrisSchinnerl — APPROVED** [review]

File: api/handlers.go (hunk 3 above)
Comment (ro-tex) [reply api/handlers.go:269]: Yes, I generally agree with that as well. In this case it felt unnecessary but maybe I'm just being lazy. I'll add it with the next PR.

**Pull request merged**